graphics/libraqm/DETAILS

```diff
@@ -1,12 +1,12 @@
           MODULE=libraqm
-         VERSION=0.7.1
-          SOURCE=raqm-$VERSION.tar.gz
+         VERSION=0.9.0
+          SOURCE=raqm-$VERSION.tar.xz
 SOURCE_DIRECTORY=raqm-${VERSION}
```

Comment:
Delete this. The tarball extracts out to;  $Some-$Pathlibraqm-0.9.0

As such a SOURCE_DIRECTORY isn't needed because MODULE and VERSION match the extraction path.